testproject/tests/test_second_step_authentication.py

```diff
@@ -470,32 +426,31 @@ def test_change_primary_method_with_backup_code(
     response = client.post(
         path="/auth/mfa/change-primary-method/",
         data={
-            "method": "sms",
+            "method": "sms_twilio",
             "code": backup_code,
         },
         format="json",
     )
     new_primary_method = active_user.mfa_methods.filter(
         is_primary=True,
     )[0]
-    assert response.status_code == 200
+    assert response.status_code == 204
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too